src/main/scala/org/apache/mesos/chronos/notification/SlackClient.scala

```diff
@@ -21,7 +21,7 @@ class SlackClient(val webhookUrl: String) extends NotificationClient {
     generator.writeStartObject()
 
     if (message.nonEmpty && message.get.nonEmpty) {
```

Comment:
I think an additional check for an error condition could not hurt. If you could add that, it would be great.